codeHF/dpl-config_run5_hf.json

```diff
@@ -518,12 +598,46 @@
         "d_selectionFlagLc": "0"
     },
     "hf-task-xic": {
-        "cutEtaCandMax": "0.8",
-        "d_selectionFlagXic": "0"
+        "cutEtaCandMax": "-1",
+        "d_selectionFlagXic": "0",
+        "pTBins": {
+            "values": [
+                "1.",
+                "2.",
+                "3.",
+                "4.",
+                "5.",
+                "6.",
+                "7.",
+                "8.",
+                "10.",
+                "12.",
+                "16.",
+                "24.",
+                "36."
+            ]
+        }
     },
     "hf-task-xic-mc": {
-        "cutEtaCandMax": "0.8",
-        "d_selectionFlagXic": "0"
+        "cutEtaCandMax": "-1",
+        "d_selectionFlagXic": "0",
+        "pTBins": {
+            "values": [
+                "1.",
+                "2.",
+                "3.",
+                "4.",
+                "5.",
+                "6.",
+                "7.",
+                "8.",
+                "10.",
+                "12.",
+                "16.",
+                "24.",
+		"36." 
+            ]
```

Comment:
something wrong with the indentation here

Reply:
Is there any helper like Clang/latest? If yes, it will be helpful to reduce the mistake.

Reply:
As reported by the Space checker, you are using tabs for indentation which might make these mistakes invisible in your editor. As advised in the Instruction for developers, you should fix this in your editor settings. If you use only spaces, you should see these mistakes directly while viewing the file. There is no formatting tool for JSON formatting unfortunately.